README.md

```diff
@@ -1,31 +1,116 @@
+![banner](https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/resources/banner.svg)
```

Comment:
Have to replace this with a `png` or run into this build error:
`ERROR  SVGs are restricted in README.md; please use other file image formats, such as PNG: https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/resources/banner.svg`.

I will fix when I get back in a couple of hours.